packages/docs-worklets/scripts/build-og-images.jsx

```diff
@@ -0,0 +1,76 @@
+import { createWriteStream } from 'fs';
+import { pipeline } from 'stream';
+import { promisify } from 'util';
+import path from 'path';
+import fs from 'fs';
+import OGImageStream from './og-image-stream';
+const { globSync } = require('glob');
+
+const getDocsMarkdownHeader = (path) => {
+  const content = fs.readFileSync(path, 'utf-8');
+  const headers = content
+    .split('\n')
+    .map((line) => line.trim())
+    .filter((line) => line.startsWith('#'))
+    .map((line, index) => ({
+      level: line.match(/#/g).length,
+      title: line.replace(/#/g, '').trim(),
+      index,
+    }))
+    .sort((a, b) => a.level - b.level || a.index - b.index);
+
+  return headers[0]?.title || 'React Native Reanimated';
```

Comment:
```suggestion
  return headers[0]?.title || 'React Native Worklets';
```

it was forgotten or should we leave it as Reanimated?

Reply:
I forgot about it 😢